build-utils/base_rebuild_single_docker_image.sh

```diff
@@ -19,21 +19,21 @@ shift
 dockerfile=$1
 echo Rebuilding docker image. Parameters: "$@"
 
-input_sha1=$( \
-  find "$0" "$@" -type f -printf "\"%p\"\n" | \
+input_sha=$( \
+  find "$0" "$@" -type f | awk '{print "\042" $1 "\042"}' | \
```

Comment:
is there a specific reason for changing the print function to awk?

Reply:
Yes, the printf option of find is not supported on all derivatives of unix. It is especially not supported by the one used as base for mac os. Hence, on mac os the script crashed.